### ISSUE TITLE
GQL "search" param does not work for default user photo field

### ISSUE THREAD

**Issue by FreekVR** [craftcms/cms#6083]
### Description
I'm trying to present an overview of user photos in the frontend, to that end I'm trying to query
only users who have a photo set.

However, the following solution, which works for custom asset fields, doesn't work for user photos and always gives 0 results.
```gql
query Photos {
  users(limit: 100, search: "photo:*") {
    photo {
      url
    }
  }
}
```

If I add a custom asset field (in this example named `customPhoto`)  to the user profile, the search does work:
```gql
query Photos {
  users(limit: 100, search: "customPhoto:*") {
    photo {
      url
    }
  }
}
```

### Additional info

- Craft version: 3.4.16
- PHP version: 7.2
- Database driver & version: n/a
- Plugins & versions: n/a

**Comment by brandonkelly** [comment]
User photos aren’t search keywords on users. We can look into adding a new `hasPhoto` element query param / GraphQL query argument, for Craft 3.5.

**Comment by andris-sevcenko** [comment]
This has been merged into the 3.5 branch.

**Comment by FreekVR** [comment]
Awesome, really appreciate the quick turnaround on these issues guys :) <3
